ProcessesApi/V1/Gateways/SoleToJoint/PersonDbGateway.cs

```diff
@@ -28,6 +28,19 @@ public PersonDbGateway(IDynamoDBContext dynamoDbContext, ILogger<PersonDbGateway
             _updater = updater;
         }
 
+        private JsonSerializerOptions GetJsonSerializerOptions()
```

Comment:
If this is applicable throughout, perhaps take out in a helper class to allow for re-use